src/model/ConfigPreset.ts

```diff
@@ -259,6 +264,7 @@ export interface NodePreset extends DockerServicePreset, Partial<NodeConfigPrese
     host?: string;
     roles?: string;
     friendlyName?: string;
+    excludeFromNemesis: boolean;
```

Comment:
why is this option needed?

Reply:
One use case,

When symbol-networks creates the 'genesis block' node (creating the network nemesis block), all the balances and link transactions are explicitly provided by the tool (according to the network layout defined in a previous step). I don't want the nemesis block node to be included at all in the nemesis block.

Reply:
This step: https://hackmd.io/Ak7Sl1A6SkKV7ga0QW00LA#Nemesis-generation

Reply:
what is this use case? where did this requirement come from?

Reply:
It's a helper ffeature for my symbol-network tool

Reply:
I see the ``excludeFromNemesis`` property getting use [here.](https://github.com/symbol/symbol-testnet/blob/deb438f6bf5114136af6b89b573c59f2f70d6e33/joeynet/nemesis-target/preset.yml#L775) 
Is this just to create local nodes?

Reply:
Yes, it's used by the tool in one of the steps. The step is when the local "nemesis" node is created. That nemesis node generates the nemesis block where all the other final node balances and transactions are provided. I don't want to include anything to that node in the nemesis block so the exclude flag.

Reply:
are you saying that there's a local node that is generating all the other nodes?

Reply:
The local node's nemesis block (seed folder) is copied over the other nodes when they are generated.